app/errors.py

```diff
@@ -0,0 +1,15 @@
+
+class VaccineError(Exception):
+    """Vaccine error"""
```

Comment:
these comments provide no additional info since class name is pretty self-explanatory